package.json

```diff
@@ -46,7 +46,6 @@
     "babel-preset-env": "^1.6.1",
     "babel-preset-react": "^6.24.1",
     "clipboard": "^2.0.0",
-    "cssnano": "^3.10.0",
```

Comment:
It looks like you need to rebase on master. Your diff shouldn't show changes outside the pages you've explicitly modified.

Reply:
Not sure why.

* I always git fetch before making changes.
* I only git added the relevant files (template and yaml), definitely not this.

Reply:
A `git fetch` only updates the remote branches, not locals (which are tracking remotes). After a fetch you can try switching to `master` and then run `git reset --hard origin/master` for example to bring it up to date. Or just switch to `master` and run `git pull` which does a fetch and merge for you.